google/auth/aws.py

```diff
@@ -690,7 +695,7 @@ def __init__(
                 "regional_cred_verification_url"
             )
 
-            # Get the environment ID. Currently, only one version supported (v1).
+            # Get the environment ID, i.e. "aws1". Currently, only one version supported (1).
```

Comment:
Is this change from `v1` to `1` just want to clarify we are using the `env_version` literally `1`?

Reply:
Correct, the value we expect for environment ID is "aws1"